src/main/java/com/bwell/auth/model/Client.java

```diff
@@ -0,0 +1,40 @@
+package com.bwell.auth.model;
+
+public class Client {
+    private String url;
```

Comment:
can we rename `url` to `auth_server_url`?

Reply:
We can. In Java, best practice is camCase; so, authServerUrl.

Reply:
Updated